config.py

```diff
@@ -596,13 +609,13 @@
 
 ica_decim = None
 
-
 # ``default_reject_comps_factory`` : callable
 #    A factory function that returns a default rejection component dictionary:
 #    A dictionary that specifies the indices of the ICA components to reject
 #    for each subject. For example you can use:
 #    rejcomps_man['subject01'] = dict(eeg=[12], meg=[7])
```

Comment:
I find this system of rejcomps_man and json file will **not** convenient to use.

what would you suggest?

Reply:
Yes, the JSON was really just an idea that I wanted to develop further at some point, but I agree currently it's sort of useless / inconvenient.

`rejcomps_man` is what existed in the pipeline already; I wanted to think about hot to improve this (i.e., noting down which ICs to reject) in a followup PR